packages/docusaurus-theme-search-algolia/src/theme/SearchBar/index.tsx

```diff
@@ -221,6 +222,149 @@ function DocSearch({
     description: 'The ARIA label and placeholder for search button',
   });
 
+  const translatedModal: DocSearchTranslations = {
+    modal: {
+      searchBox: {
+        resetButtonTitle: translate({
+          id: 'theme.SearchModal.searchBox.resetButtonTitle',
+          message: 'Clear the query',
+          description: 'The label for search box reset button',
+        }),
+        resetButtonAriaLabel: translate({
+          id: 'theme.SearchModal.searchBox.resetButtonAriaLabel',
+          message: 'Clear the query',
+          description: 'The ARIA label for search box reset button',
+        }),
+        cancelButtonText: translate({
+          id: 'theme.SearchModal.searchBox.cancelButtonText',
+          message: 'Cancel',
+          description: 'The label for search box cancel button',
+        }),
+        cancelButtonAriaLabel: translate({
+          id: 'theme.SearchModal.searchBox.cancelButtonAriaLabel',
+          message: 'Cancel',
+          description: 'The ARIA label for search box cancel button',
+        }),
```

Comment:
Looks repetitive to me. What are the odds that the user wants different translations for these labels?

Reply:
OK. I'll change it to one